app/access/api.py

```diff
@@ -66,3 +71,26 @@ def create_user(request: HttpRequest, user_in: UserSchema) -> UserSchema:
         provider=provider
     )
     return user_to_response(user_out)
+
+
+@router.delete("users/{username}")
+def delete(request: HttpRequest, username: str) -> HttpResponse:
+    """
+    Delete the user with the given username.
+
+    Return HTTP status code
+    - 204 (No Content) if the user has been deleted
+    - 404 (Not Found) if there is no user with the given username
+    - 500 (Internal Server Error) if there is inconsistency with cognito
+    - 503 (Service Unavailable) if cognito cannot be reached
+    """
+
+    with transaction.atomic():
+        model = User.objects.select_for_update().filter(username=username).first()
+        if not model:
+            raise Http404("Not Found")
+        deleted = delete_user(model)
+        if not deleted:
+            raise HttpError(500, "Internal Server Error")
+        model.delete()
```

Comment:
Did we not want to deactivate a user on delete?

Reply:
Yes, in a second step. There is currently none of the required fields available. If I remember correctly, this is a separate ticket.

Reply:
As discussed in person: this is covered in PB-1070